src/sentry/conf/server.py

```diff
@@ -2092,7 +2092,12 @@ def create_partitioned_queues(name):
 MIGRATIONS_TEST_MIGRATE = os.environ.get("MIGRATIONS_TEST_MIGRATE", "0") == "1"
 # Specifies the list of django apps to include in the lockfile. If Falsey then include
 # all apps with migrations
-MIGRATIONS_LOCKFILE_APP_WHITELIST = ()
+MIGRATIONS_LOCKFILE_APP_WHITELIST = (
+    "jira_ac",
+    "nodestore",
+    "sentry",
+    "social_auth",
+)
```

Comment:
I never really thought about doing this, but I guess it makes sense - we're not going to get migration conflicts from migrations in 3rd party libraries